src/odemis/acq/stream/_projection.py

```diff
@@ -41,6 +45,8 @@ def __init__(self, stream):
         stream (Stream): the Stream to project
         '''
         self.stream = stream
+        self.name = stream.name
```

Comment:
Sure. I think I did this because there were many callers which expected a stream but got a projection, and dind't know how to deal with the name. So this was the easy way to patch the issue.